integration-tests/cypress/e2e/project/deck/deck.cy.ts

```diff
@@ -0,0 +1,300 @@
+      new Explorer().getTreeItemByLabel('renamed_New Daily Representation').should('exist');
+
+      cy.getByTestId('representation-tab-renamed_New Daily Representation').should('exist');
+      new Explorer().rename('renamed_New Daily Representation', 'New Daily Representation');
+    });
+
+    it('We remove the Deck Representation on the Daily project', () => {
+      new Explorer().getTreeItemByLabel('New Daily Representation').should('exist');
+      new Deck().getDeckRepresentation().should('exist');
+      new Explorer().delete('New Daily Representation');
+      new Explorer().getTreeItemByLabel('New Daily Representation').should('not.exist');
+      cy.getByTestId('representation-area').find('h5').should('have.text', 'The Deck does not exist anymore');
+    });
+  });
+
+  context('We now verify the Deck representation refresh', () => {
+    let taskProjectId: string;
+    before(() => {
+      createTaskProjectAndDeckRepresentation('Daily Project Dev', 'New Daily Representation').then(() => {
+        cy.url().then(($url) => {
+          const prefix = Cypress.config().baseUrl + '/projects/';
+          taskProjectId = $url.substring(prefix.length, $url.indexOf('/', prefix.length + 1));
+        });
+      });
+    });
+
+    beforeEach(() => {
+      openDeckRepresentation(taskProjectId, 'Daily Project Dev', 'New Daily Representation');
+    });
+
+    after(() => {
+      cy.deleteProject(taskProjectId);
+    });
+    it('We create a new daily tag to verify that the deck is properly refreshed', () => {
+      cy.getByTestId('Daily Project Dev-more').click();
+      cy.getByTestId('new-object').click();
+      cy.getByTestId('create-object').should('be.enabled');
+      cy.getByTestId('childCreationDescription').click();
+      cy.getByTestId('childCreationDescription').get('[data-value="Tag"]').should('exist').click();
+      cy.getByTestId('create-object').click();
+      cy.getByTestId('::').click();
+      cy.getByTestId('input-Prefix').type('daily{enter}');
+      cy.getByTestId('input-Suffix').type('Today{enter}');
+      new Deck().getLane('Today').should('exist');
+    });
+
+    it('We set the task description and verify that the card is refreshed.', () => {
+      const deck = new Deck();
+      const explorer = new Explorer();
+      deck.getCard('Monday', 'Idea').should('exist');
+      explorer.getTreeItemByLabel('Idea').click();
+      cy.getByTestId('input-Description').type('The Description{enter}');
+      deck.getCard('Monday', 'Idea').contains('The Description');
+    });
+
+    it('We change a task tag to verify that the deck is properly refreshed (the card should be removed from the lane)', () => {
+      const deck = new Deck();
+
+      deck.getCard('Monday', 'Idea').should('exist');
+      cy.getByTestId('Idea').click();
+      cy.getByTestId('reference-value-daily::Monday').find('svg').click();
+      cy.getByTestId('card-Idea').should('not.exist');
```

Comment:
In this case, it's better to create a function to mutualize the behavior of a widget
Something like 
```typescript
new Explorer().select('Idea');
new Details().deleteReferenceWidgetOption('Tags', 'daily::Monday');
deck.getCard('Monday', 'Idea').should('not.exist');
```
By adding to Details.ts
```typescript
  public deleteReferenceWidgetOption(referenceWidgetLabel: string, optionLabel: string): void {
    this.getReferenceWidget(referenceWidgetLabel).findByTestId(`reference-value-${optionLabel}`).find('svg').click();
  }
```